docs/ko/getting-started.md

```diff
@@ -1,6 +1,8 @@
 # 시작하기
 
-es-git은 Node.js용 현대적인 Git 라이브러리예요. 간편한 인터페이스로 복잡한 Git 작업도 쉽게 통합할 수 있으며, TypeScript 타입을 지원해서 빠르고 안정적인 개발이 가능해요.
+N-API를 기반으로 구축되어 설치 속도가 매우 빠르고 안정성이 뛰어난 Node.js용 최신 Git 라이브러리에요.
```

Comment:
```suggestion
es-git은 N-API를 기반으로 구축되어 설치 속도가 매우 빠르고 안정성이 뛰어난 Node.js용 최신 Git 라이브러리에요.
```
Thanks for your contribution! I think it would be nice add some explanations for library.

Reply:
Thanks for good suggestsion